M2/Macaulay2/e/f4/f4.hpp

```diff
@@ -68,6 +68,10 @@
 //      Hilbert function use
 //
 
+#include "interface/computation.h"
+#include "interface/mutable-matrix.h"
+
+#include "comp.hpp"
```

Comment:
@mahrud perhaps we need to be more careful here about where files are, since there are possible name collisions we need to consider.  e.g. in this file, should this be `#include "f4-types.hpp"` or `#include "f4/f4-types.hpp"`?

Reply:
I thought that was the point of putting `f4-` in the beginning of the name. I would be okay with moving `f4-* -> f4/*` in the future.

Reply:
Well, they are in a subdirectory, so the issue of having the same name files in different directories in e is something to worry about.  Again, currently, it is not an issue for this pull request 9as long as there are no name conflicts/ambiguities, but in the next round of changes, I want to be consistent with include file names.